### PR TITLE
description PL EN UA added

### PR DESCRIPTION
**Pull request by AleksanderOrtenburger**
description PL EN UA added to listings and detailed view card

### CODE REVIEW

File: src/views/notices/View_Notice/cards/Card_HelpOffer.jsx [hunk 1]
```diff
@@ -58,7 +61,9 @@ const HelpOfferCard = () => {
     <Card>
       <div className="flex flex-col md:flex-row">
         <div className="flex-1">
-          {!!description && <Item label="Opis:" value={description} />}
+        {!!description && <Item label={t('common.opis')} value={descriptionPL} />}
+          {!!description && <Item label={t('common.opisUA')} value={descriptionUA} />}
```
Comment (sem-yura) [inline src/views/notices/View_Notice/cards/Card_HelpOffer.jsx]: should be !!descriptionUA

File: src/views/notices/View_Notice/cards/Card_HelpOffer.jsx [hunk 2]
```diff
@@ -58,7 +61,9 @@ const HelpOfferCard = () => {
...
+          {!!description && <Item label={t('common.opisEN')} value={descriptionEN} />}
```
Comment (sem-yura) [inline src/views/notices/View_Notice/cards/Card_HelpOffer.jsx]: descriptionEN

File: src/views/notices/View_Notice/cards/Card_HelpOffer.jsx (hunk 1 above)
Comment (AleksanderOrtenburger) [reply src/views/notices/View_Notice/cards/Card_HelpOffer.jsx]: If descriptionUA and descriptionEN are translated descrription in the backend , so if there is no description there won't be descriptionUA,EN etc

Comment (sem-yura) [reply src/views/notices/View_Notice/cards/Card_HelpOffer.jsx]: You could join these fields and use one check

**Comment by OlehKhozhai** [comment]
please pull the last dev changes
and take a look at how currently it is working
I removed all cards folder

File: src/views/notices/components/common/Fields/Description.jsx [hunk 3]
```diff
@@ -3,17 +3,36 @@ import { useTranslation } from 'react-i18next'
...
+        valueClassName={`text-gray-800 font-normal `}
+      />
+
+      <NoticeDetailsItem
+        label={descriptionEN && t('common.opisEN')}
+        className="flex-col"
+        icon={<Icon.MdOutlineTextSnippet {...iconConfig} />}
+        value={descriptionEN}
+        valueClassName={`text-gray-800 font-normal `}
+      />
```
Comment (OlehKhozhai) [inline src/views/notices/components/common/Fields/Description.jsx:34]: how about adding additional check if descriptionEN or descriptionPL or descriptionUA is `undefined`
```
descriptionEN && <NoticeDetailsItem
        label={descriptionEN && t('common.opisEN')}
        className="flex-col"
        icon={<Icon.MdOutlineTextSnippet {...iconConfig} />}
        value={descriptionEN}
        valueClassName={`text-gray-800 font-normal `}
      />
```

**Review by MikeRock — APPROVED** [review]

**Pull request merged**